src/atwho.js

```diff
@@ -29,11 +33,23 @@ define([
     var cachedMugData = timed(function(form) {
         return _.chain(form.getMugList())
                 .map(function(mug) {
+                    // probably better to use text-overflow: ellipsis
```

Comment:
Didn't quite get the text-overflow css to work properly. (never actually cut off the data)